packages/maker/wix/src/MakerWix.ts

```diff
@@ -38,6 +38,10 @@ export default class MakerWix extends MakerBase<MakerWixConfig> {
       outputDirectory: outPath,
     }) as MSICreatorOptions);
 
+    if (this.config.beforeCreate) {
+      await this.config.beforeCreate(creator);
```

Comment:
`await Promise.resolve(this.config.beforeCreate(creator));`

That way if `beforeCreate` doesn't return a promise it won't blow up 👍